core/standard/clause_9_sorting.adoc

```diff
@@ -18,24 +18,27 @@ handling missing values; specifying a high value indicating that the correspondi
 
 include::requirements/sorting/REQ_sortby-response.adoc[]
 
-include::requirements/sorting/REQ_sortables-op.adoc[]
+include::requirements/sorting/REQ_get-sortables-op.adoc[]
 
-include::requirements/sorting/REQ_sortables-success.adoc[]
+include::requirements/sorting/REQ_get-sortables-success.adoc[]
+
+The response is returned as a JSON Schema document that describes a single JSON object where each property is a sortable. Note that the sortables schema does not specify a schema of any object that can be retrieved from the API. JSON Schema is used for the sortables to have a consistent approach for describing schema information and JSON Schema is/will be used in other parts of OGC API Features to describe schemas for GeoJSON feature content including in OpenAPI documents.
+
+NOTE: The OGC Feature API Standards Working Group has a schema harmonization task that could lead to a future OGC API standard that will deprecate the approach for the sortables resource specified in this document.
```

Comment:
`s/OGC Feature API Standards Working Group/OGC API - Features Standards Working Group/g` (sorry, I am fresh off a deep editorial sweep of the standard).

Reply:
The name of the group is "Features API SWG", so the current text is *almost* correct (an "s" is missing). I will add it.

Reply:
... here is the link to the group: https://www.ogc.org/projects/groups/featapiswg

Reply:
OK.  Looks like the other OGC API Standards used the `OGC API - xxx` convention for their SWG names.  All good/carrying on.